Tools/jit/_stencils.py

```diff
@@ -47,6 +48,68 @@ class HoleValue(enum.Enum):
     ZERO = enum.auto()
 
 
+_PATCH_FUNCS = {
+    # aarch64-apple-darwin:
+    "ARM64_RELOC_BRANCH26": "patch_aarch64_26r",
+    "ARM64_RELOC_GOT_LOAD_PAGE21": "patch_aarch64_21rx",
+    "ARM64_RELOC_GOT_LOAD_PAGEOFF12": "patch_aarch64_12x",
+    "ARM64_RELOC_PAGE21": "patch_aarch64_21r",
+    "ARM64_RELOC_PAGEOFF12": "patch_aarch64_12",
+    "ARM64_RELOC_UNSIGNED": "patch_64",
+    # x86_64-pc-windows-msvc:
+    "IMAGE_REL_AMD64_REL32": "patch_x86_64_32rx",
+    # aarch64-pc-windows-msvc:
+    "IMAGE_REL_ARM64_BRANCH26": "patch_aarch64_26r",
+    "IMAGE_REL_ARM64_PAGEBASE_REL21": "patch_aarch64_21rx",
+    "IMAGE_REL_ARM64_PAGEOFFSET_12A": "patch_aarch64_12",
+    "IMAGE_REL_ARM64_PAGEOFFSET_12L": "patch_aarch64_12x",
+    # i686-pc-windows-msvc:
+    "IMAGE_REL_I386_DIR32": "patch_32",
+    "IMAGE_REL_I386_REL32": "patch_x86_64_32rx",
+    # aarch64-unknown-linux-gnu:
+    "R_AARCH64_ABS64": "patch_64",
+    "R_AARCH64_ADD_ABS_LO12_NC": "patch_aarch64_12",
+    "R_AARCH64_ADR_GOT_PAGE": "patch_aarch64_21rx",
+    "R_AARCH64_ADR_PREL_PG_HI21": "patch_aarch64_21r",
+    "R_AARCH64_CALL26": "patch_aarch64_26r",
+    "R_AARCH64_JUMP26": "patch_aarch64_26r",
+    "R_AARCH64_LD64_GOT_LO12_NC": "patch_aarch64_12x",
```

Comment:
```          case HoleKind_ARM64_RELOC_GOT_LOAD_PAGEOFF12:
            case HoleKind_ARM64_RELOC_PAGEOFF12:
            case HoleKind_IMAGE_REL_ARM64_PAGEOFFSET_12A:
            case HoleKind_IMAGE_REL_ARM64_PAGEOFFSET_12L:
            case HoleKind_R_AARCH64_ADD_ABS_LO12_NC:
            case HoleKind_R_AARCH64_LD64_GOT_LO12_NC:
```

used to all be in the same case and now have a mixture of this function and `patch_aarch64_12` used for these instructions. Is this intentional?

Reply:
Ditto.